docs/pages/push-notifications/using-fcm.md

```diff
@@ -65,9 +65,15 @@ In order for Expo to send notifications from our servers using your credentials,
 
 3. Copy the token listed next to **Server key**.
 
-> **Note:** Server Key is only available in **Cloud Messaging API (Legacy)**, which may be Disabled by default. Enable it by clicking the 3-dot menu > Manage API in Google Cloud Console and follow the flow there. Once the legacy messaging API is enabled, you should see Server Key in that section.
+  > **Note:** Server Key is only available in **Cloud Messaging API (Legacy)**, which may be Disabled by default. Enable it by clicking the 3-dot menu > Manage API in Google Cloud Console and follow the flow there. Once the legacy messaging API is enabled, you should see Server Key in that section.
 
-4. Run `expo push:android:upload --api-key <your-token-here>`, replacing `<your-token-here>` with the string you just copied. We'll store your token securely on our servers, where it will only be accessed when you send a push notification.
+4. In your [expo account](https://expo.dev/), select your project, and click on Credentials in the navigation menu. Click on your Application Identifier _(something like com.yourcompany.yourapp)_. 
```

Comment:
These points are not coming up with as number 4 or number 5. I think adding the note in between might be breaking the flow of numbered list. Can you please have a look to fix this?

<img width="885" alt="CleanShot 2022-09-29 at 04 21 17@2x" src="https://user-images.githubusercontent.com/10234615/192902344-ef9f9f71-873b-4f81-8920-bc6dc9e3b315.png">